node-tests/addon-test.js

```diff
@@ -691,7 +691,7 @@ describe('ember-cli-babel', function() {
       expect(pluginRequired).to.be.true;
     });
 
-    it('returns false when targets do not require plugin', function() {
+    it.only('returns false when targets do not require plugin', function() {
```

Comment:
@stefanpenner sorry about that... was (foolishly) not even considering anyone else using `isPluginRequired` when we made string version target changes (originally targeted for 2.x) :(